src/main/java/net/barrage/school/java/ecatalog/app/ProductServiceImpl.java

```diff
@@ -29,6 +34,10 @@ public List<Product> listProducts() {
 
     @SneakyThrows
     @Override
+    @Cacheable(
```

Comment:
Hmm... I think in real world for this type of API such caching can be useless) But we need to have stats from this API to decide ofc.